server.js

```diff
@@ -426,11 +432,17 @@ app
             IAtitle.trim() !== ""
               ? replaceTitle(IAtitle.trim())
               : replaceTitle(await getPDLTitle(options));
+          const isAlphanumericLess50 = /^(?![0-9]+$)[a-zA-Z0-9]{1,50}$/;
```

Comment:
Please define regex once and use it everywhere. Don't define it multiple times.

Reply:
@coderwassananmol  fixed